Service/GetAdminPaymentSessionData.php

```diff
@@ -82,11 +85,19 @@ public function execute(string $orderId, ?array $customData = null): array
                 $payload['transaction']['custom_data'] = base64_encode($this->serializer->serialize($customData));
             }
 
+            $payload['custmoer'] = [];
```

Comment:
- [ ] `custmoer` -> `customer`

Reply:
yep, this is a typo

Reply:
@adamcox-acquired this was fixed in beta5 version:
https://github.com/stefane2buy1click/acquired-payments-magento/commit/b0ed3b20239b0c732b9493ad849ad382690475b0